packages/addons/addon-depends/icu/package.mk

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="icu"
+PKG_VERSION="67-1"
+PKG_SHA256="571d3723a8055d85e513b19b70b4f7f370da10df084c3bb598e0fb8333da0a16"
+PKG_LICENSE="https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE"
```

Comment:
```suggestion
PKG_VERSION="67.1"
PKG_SHA256="8987b530d7954e9a1b8c041dcb8959221d8a794710302a26e800174d2d511ad2"
PKG_LICENSE="MIT"
```

Reply:
I am not sure that [this](https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE) is a MIT license

Reply:
its MIT with additions that fall completely out of the standard, so tbh I have no real idea
`PKG_LICENSE="Custom"` might be the alternative

Reply:
Done